ffi/airgap.go

```diff
@@ -75,6 +75,7 @@ func PullFromAirgap(socket, licenseData, airgapDir, downstream, outputFile, regi
 			Downstreams:         []string{downstream},
 			LocalPath:           releaseDir,
 			LicenseFile:         licenseFile,
+			AirgapMetaFile:      filepath.Join(airgapDir, "airgap.yaml"),
```

Comment:
@divolgin thinking about this and changes in the future... should we just walk the filetree of the archive, matching GVK instead?

Reply:
@marccampbell this file is never in a subdirectory.... I think if we list files in the root of the bundle that should be sufficient.